unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuildModeHUD/BuildModeHUDView.cs

```diff
@@ -7,7 +7,6 @@
 
 public class BuildModeHUDView : MonoBehaviour
 {
-
     public SceneLimitInfoController sceneLimitInfoController;
     public SceneObjectCatalogController sceneObjectCatalogController;
     public ToolTipController toolTipController;
```

Comment:
tooltip is a single word, thus we should name this variable `tooltipController` instead of `toolTipController` and rename the class too.